Prox/Prox/PlaceCarouselViewController.swift

```diff
@@ -206,3 +219,15 @@ extension PlaceCarouselViewController: CLLocationManagerDelegate {
         print("lol-location \(error.localizedDescription)")
     }
 }
+
+
+private extension Date {
```

Comment:
I've moved to use `>`. Even if we kept this, I would prefer to keep the extension private unless we know it is going to be used elsewhere.